tests/helper/helper.go

```diff
@@ -456,7 +464,8 @@ func WaitForDeploymentReplicaReadyCount(t *testing.T, kc *kubernetes.Clientset,
 
 // Waits until statefulset count hits target or number of iterations are done.
 func WaitForStatefulsetReplicaReadyCount(t *testing.T, kc *kubernetes.Clientset, name, namespace string,
-	target, iterations, intervalSeconds int) bool {
+	target, iterations, intervalSeconds int,
+) bool {
```

Comment:
Is this new line intended or just auto format?

Reply:
yeah gofmt, I put them onto a single line, but happy to format them how you'd prefer